packages/node_modules/@node-red/nodes/core/common/60-link.html

```diff
@@ -76,7 +92,18 @@
                     RED.view.redraw();
                 }
             });
-        const candidateNodes = RED.nodes.filterNodes({type:targetType});
+        const candidateNodes = RED.nodes.filterNodes({type:targetType}).filter(n => {
+            console.log(node);
+            console.log(n);
+            if (n.visibility === "none") {
```

Comment:
What is the use case for "none". If you can see the link how do you link to it ?

Reply:
If you have another in or out node that you want to connect to but for all other links basically forget about this one, this can be used.

Reply:
So you have to connect it up first then hide it ?

Reply:
No, you will connect to another node from the hidden one, but the hidden node will not appear in any list to connect to it from any other node.

Reply:
ah - ok... can see that now.

Reply:
The idea behind that was to have a tab that has a lot of IO nodes to be reused in different locations. They would interact with links. Once you do that, you will end up with a big ist of link that you need to look through when connecting them. With this feature, you could hide "unrelevant" links completely.